schema-loader/src/integration-test/java/com/scalar/db/schemaloader/SchemaLoaderIntegrationTestBase.java

```diff
@@ -420,27 +420,8 @@ private void deleteTables_ShouldDeleteTablesWithCoordinator() throws Exception {
     assertThat(transactionAdmin.coordinatorTablesExist()).isFalse();
   }
 
-  private int executeCommandWithArgs(List<String> args) throws Exception {
-    List<String> command =
-        ImmutableList.<String>builder()
-            .add("java")
-            .add("-jar")
-            .add(schemaLoaderJarPath)
-            .addAll(args)
-            .build();
-    ProcessBuilder processBuilder = new ProcessBuilder(command);
-
-    Process process = processBuilder.start();
-    try (final BufferedReader input =
-        new BufferedReader(
-            new InputStreamReader(process.getErrorStream(), Charset.defaultCharset()))) {
-      String line;
-      while ((line = input.readLine()) != null) {
-        System.out.println(line);
-      }
-    }
-
-    return process.waitFor();
+  private int executeWithArgs(List<String> args) {
+    return SchemaLoader.mainInternal(args.toArray(new String[0]));
```

Comment:
Previously we run the integration tests with the fat jar, so we need to build the fat jar before running the integration tests. But after this change, we directly call `SchemaLoader.mainInternal()` to run the schema loader, so we no longer need to build the fat jar.